spec/controllers/idv/document_capture_controller_spec.rb

```diff
@@ -191,14 +197,41 @@
       end
     end
 
-    context 'socure is the default vendor but facial match is required' do
+    context 'socure is the default vendor' do
       let(:idv_vendor) { Idp::Constants::Vendors::SOCURE }
-      let(:facial_match_required) { true }
 
-      it 'does not redirect to Socure controller' do
-        get :show
+      describe 'facial match is required' do
+        let(:facial_match_required) { true }
+
+        it 'does not redirect to Socure controller' do
+          get :show
+
+          expect(response).to_not redirect_to idv_socure_document_capture_url
+        end
+      end
+
+      describe 'facial match not required but socure user limit reached' do
+        before do
+          allow(IdentityConfig.store).to receive(:doc_auth_socure_max_allowed_users).and_return(1)
+          Idv::SocureUserSet.new.add_user!(user_uuid: '001')
+        end
```

Comment:
```suggestion
        end
        
```
(total nit, but I obsess over whitespace)